.gitignore

```diff
@@ -1,2 +1,3 @@
 *.DS_Store
 **/qsite/*
+**/updates/*.scpt
```

Comment:
make sure this only catches the vscode generated files and doesn't ignore anything from the main line.